### PR TITLE
Rendering News Articles

### PR DESCRIPTION
**Pull request by calvincurry22**
**Changes**

- Added functionality to render the articles list to the UI

**Testing** 

- Serve the application to localhost
- Click the Add new Article Button
- Complete the form and click Save Article
- Ensure that the new Article card renders to the UI
- Click the link in the Article card to ensure that it opens the web page a new tab

### CODE REVIEW

**Review by iparkerhodge — APPROVED** [review]
👍 
-delete works
-add article works
-"http://" alert works on url input

**Review by Keartroth — APPROVED** [review]
Everything functions. Articles, events, and messages all add. Merge approved.

**Review by durrjp — APPROVED** [review]
All components are functioning correctly for articles.

File: src/scripts/articles/ArticleList.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+let contentTarget = document.querySelector(".articlesContainer")
+const eventHub = document.querySelector("#container")
+
+
+export const ArticleList = () => {
+
+    const articleCollection = useArticles()
+
+    articleCollection.map(articleObj => {
+        contentTarget.innerHTML += Article(articleObj)
```
Comment (stevebrownlee) [inline src/scripts/articles/ArticleList.js:18]: Too expensive. Build the string, join it, and put the entire string in the DOM

File: src/scripts/articles/ArticlesProvider.js [hunk 2]
```diff
@@ -0,0 +1,48 @@
+// Author: Calvin Curry
+// This module is responsible for providing the parsed article data from the database.JSON file. 
+
+
+const eventHub = document.querySelector("#container")
+
+
+const dispatchSavedArticle = () => {
```
Comment (stevebrownlee) [inline src/scripts/articles/ArticlesProvider.js:8]: Too specific. Articles can also be deleted. This event should simply say that the state changed.

**Pull request merged**